docs/tech/tech-index.md

```
@@ -0,0 +1,20 @@
+#index #tech
+
+1. [nodejs](nodejs.md)
+2. [npm](npm.md)
+3. [expressjs](expressjs.md)
+	1. middlewares
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)